data/conditions.ts

```diff
@@ -269,15 +269,17 @@ export const Conditions: {[k: string]: ConditionData} = {
 		duration: 2,
 		onStart(target, source, effect) {
 			this.effectState.move = effect.id;
+			let slot: PokemonSlot | null = target.getSlot();
```

Comment:
Why is this `| null`? `getSlot` always returns a `PokemonSlot`.

Reply:
just in case the move is called by a sub-move

Reply:
Is the type signature for `getSlot()` not correct?
```ts
getSlot(): PokemonSlot {
	const positionOffset = Math.floor(this.side.n / 2) * this.side.active.length;
	const positionLetter = 'abcdef'.charAt(this.position + positionOffset);
	return (this.side.id + positionLetter) as PokemonSlot;
}
```